docs/api.rst

```diff
@@ -110,6 +110,14 @@ Core
       C2(x=42, y=[])
 
 
+.. autofunction:: attr.singleton
+
+    This is a decorator to build singleton classes.
+
+    It will usually not be accessed directly, but it also has support for
+    transforming a :class:`~collections.namedtuple` into a singleton.
```

Comment:
I think this reference to `namedtuple` needs a sphinx plugin to point it at the python documentation. Not sure if that plugin is being used.